.github/workflows/docker-builder-devcontainer.yml

```
@@ -52,6 +52,10 @@ env:
 
 jobs:
   build_dockerfile_bazel_base:
+    if: |
+      (github.event_name == 'push' || github.event_name != 'schedule') &&
```

This can be reduced to 
```suggestion
      (github.event_name != 'schedule') &&
```
Is that what you want? Why is everything except for schedule OK?

See https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows for a full list of events

---

maybe you wanted to allow 'schedule' as trigger?
```suggestion
      (github.event_name == 'push' || github.event_name == 'schedule') &&
```

---

@maxhbr  This PR is a temporary arrangement to update the liblsan0 package on the bazel-base image. Changes in the PR will be reverted back with an upcoming PR.

---

can you maybe instead just allow the event that you want to add?

---

@maxhbr the changes are pushed as requested